packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/resolution/ResolutionFeature.java

```diff
@@ -144,6 +143,8 @@ static Size computeBestPreviewSize(int cameraId, ResolutionPreset preset)
    * @return The best possible {@link android.media.CamcorderProfile} that matches the supplied
    *     {@link ResolutionPreset}.
    */
+  @TargetApi(Build.VERSION_CODES.R)
+  @SuppressWarnings({"fallthrough", "deprecation"})
```

Comment:
We should move the comment below about why we're doing fall through up to this line so it's clear.

We should also fix the violations of https://google.github.io/styleguide/javaguide.html#s4.8.4-switch while we're here, adding a comment to the end of every `case`.

Reply:
Done